pkg/config/types.go

```diff
@@ -53,4 +53,12 @@ const (
 	// GPU
 	GPUSMUtilization  = "gpu_sm_util"
 	GPUMemUtilization = "gpu_mem_util"
+
+	// idle power calculation method
+	IdlePowerMethodAvg   = "idle_power_method_avg"   // average metrics usage
+	IdlePowerMethodMax   = "idle_power_method_max"   // max metrics usage
+	IdlePowerMethodMin   = "idle_power_method_min"   // min metrics usage
+	IdlePowerMethodCurr  = "idle_power_method_curr"  // curr metrics usage
+	IdlePowerMethodCount = "idle_power_method_count" // number of containers
```

Comment:
did we consider process here already?

Reply:
@rootfs what is this idle power calculation method?

Why would someone disable the idle power?

Is this necessary for this PR or we can discuss it in another PR so that we don't block the optimizations here?

Reply:
@marceloamaral the fixes in this PR are built together for validation. Once done, they will be submitted separately in following PRs.